package/src/layouts/Layout.astro

```diff
@@ -54,6 +58,7 @@ const { pageTitle, hideFooter, hideHeader, ogImage } = Astro.props as AstroHashn
         ]
       }}
     />
+    {useTranstions && <ViewTransitions />}
```

Comment:
I'm not sure how pages work in this integration, but you may need some client directives too on some pages? Or is it up to the user to add those?

Reply:
With this setup, its pulling data using graphql for any of the included pages so its not needed, if a user wanted to utilize this layout as their layout for other pages, they would be able to call whatever they need for their components specifically.